examples/Boost-filesystem-shared/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2013-2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+set(TESTING_CONFIG_OPT FILEPATH ${CMAKE_CURRENT_LIST_DIR}/cmake/Hunter/config.cmake) #set BUILD_SHARED_LIBS=ON
```

Comment:
In case we are using FILEPATH the path to config.cmake can be any. The current code is correct however there is no need to use `cmake/Hunter` pattern, it can be just `${CMAKE_CURRENT_LIST_DIR}/config.cmake`.